docs/en/operations/settings/merge-tree-settings.md

```diff
@@ -989,6 +989,12 @@ Used to regulate how resources are utilized and shared between mutations and oth
 
 Default value: an empty string
 
+## default_compression_codec
+
+Specifies the default compression codec to be used if none is defined for a particular column in the table declaration. If a column has a designated compression codec, it will override this default setting.
```

Comment:
```suggestion
Specifies the default compression codec to be used if none is defined for a particular column in the table declaration. If a column has a designated compression codec, it will take priority over this default setting.
```